wiki/FAQ/en.md

```diff
@@ -274,71 +358,126 @@ _Detailed explanation of scoring can be found at the game style's own respective
+
+For osu!mania, Entozer has compiled a decent list sorted by relative play difficulties. You can find it here: <https://osu.ppy.sh/forum/t/147552>
+
+Also, [the ranked beatmap version of osu! tutorial][osu!tutorial rank] is good for practicing some of the possible mapping design you will come across. It is an excellent training map for newbies and experienced players.
 
 Alternatively, you can [sort the ranked beatmap list by difficulty][sort beatmaplist difficulty]. The easiest maps are at the top of the list.
 
 #### What does Level in my profile actually do?
-Absolutely _nothing_. There is no difference in any way when comparing a Level 100 player against a Level 1 player except the player's own skills and experience of play. There is no handicap for the lower level nor there is an advantage for the higher level (except personal experience and hardware used).
+**Absolutely nothing**. There is no difference in any way when comparing a Level 100 player against a Level 1 player except the player's own skills and experience of the game mode in question. There is no handicap for the lower level nor is there an advantage for the higher level (except personal experience and hardware used).
 
-Level is based on your total score accumulated when online with your account whether you pass or fail the beatmap. [Formula given here][Score#Level wikilink].
+Level will be based on total score accumulated when online with account regardless of pass or fail on the beatmap. [Formula given here][Score#Level wikilink].
 
 #### Why is my profile stats different ingame?
 Keep in mind that osu! has four game modes so you might be looking at the wrong game mode.
 
-Manually select the game mode at your profile that you want to view.
-
 ![Profile_Mode_Buttons][Profile_Mode_Buttons image]
 
+_Manually select the game mode at your profile that you want to view._
+
+#### How do I change the default keyboard key input for my game mode of choice?
+![Options sidebar with quickicons][Options quickicons image]
+
+From the _Input_ icon in the _Options_ sidebar as shown above, under **Keyboard** section as shown below.
+
+![Options keyboard section][Options keyboard image]
+
+#### Game over? Where did I messed up?
+_Full explanation: [Replay][Replay wikilink]_
+
+Press `F1` to replay the currently failed play. To restart the replay again, hold the custom default grave/tilde (`` `/~ ``) key (can be changed) or `Ctrl`+`R` for a while. The replay ends when the screen darkens with an osu!logo in the middle, rather than the game over screen.
+
+Due to the non-standard replay screen, use `Esc` key to return to Song Selection. Retrying the play (as in, actual play) of the beatmap in this state is not possible. It currently works in Solo mode only when you are given the game over screen. **This replay will not be saved nor allowed to be exported.** Use third-party software to record and save this failed replay instead.
```

Comment:
``use the `Esc` ``
<hr/>
last sentence:

`However, you can use third-party software to record` ...